-util.js

```diff
@@ -50,6 +42,21 @@ function isElementNode (node) {
 	return !!(node && node.nodeType === 1);
 }
 
+var isConnected; 
+function setIsConnected(global) {
+	isConnected = 'isConnected' in global.Node.prototype ?
+		function(node) { return node.isConnected; } :
+		function(node) { 
+			var doc = getDocument();
+			return node === doc || contains(doc, node);
+		};
+}
+setIsConnected(globals.getKeyValue("global"));
+globals.onKeyValue("global", function(global) {
```

Comment:
While `Node` is on the `window`, the switching of the `document` is what matters.

Reply:
1.  Make a test for vdom to make sure a `non-isConnected` supported document still works.
2.  node.ownerDocument
3. Switch to listening to `document` event
4. `document.createElement("div").constructor.prototype` to test `isConnected`

Reply:
- [components](https://github.com/canjs/can-component/blob/5a1c78a8c59e6d442c5fb422c61a2c06010c135a/test/helpers.js#L22)
- [can-stache-bindings](https://github.com/canjs/can-stache-bindings/blob/e49be2938280c73828f285c1d0007158f12161c0/test/helpers.js#L9)